File: c/q14.c
// Fix the function to print a right inverted pyramid pattern with correct spacing.

void printInvertedPyramid(int rows) {
    for (int i = rows; i >= 1; i--) {
        for (int j = 0; j < rows - i; j++ {
            printf("  ");
        }
        for (int k = 0; k < i; k++) {
            printf("* ");
        }
        printf("\n");
    }
}

/*
Test case:
Input: 5
Output:
* * * * * 

  * * * * 

    * * * 

      * * 

        * 
*/

File: javascript/q14.js
// Fix the function to print a right inverted pyramid pattern with correct spacing.

function printInvertedPyramid(rows) {
    for (let i = rows; i >= 1; i--) {
        for (let j = 0; j < rows - i; j++) {
            console.log("  ");
        }
        for (let k = 0; k < i; k++) {
            console.log("* ");
        }
        console.log("\n");
    }
}

/*
Test case:
Input: 5
Output:
* * * * * 

  * * * * 

    * * * 

      * * 

        * 
*/